src/iotjs_binding_helper.c

```diff
@@ -115,7 +115,8 @@ jerry_value_t iotjs_make_callback_with_result(jerry_value_t jfunction,
                                               const iotjs_jargs_t* jargs) {
   // Calls back the function.
   jerry_value_t jres = iotjs_jhelper_call(jfunction, jthis, jargs);
-  if (jerry_value_has_error_flag(jres)) {
+  iotjs_environment_t* env = iotjs_environment_get();
+  if (jerry_value_has_error_flag(jres) && !iotjs_environment_is_exiting(env)) {
```

Comment:
Can we do the `iotjs_environment_is_exiting(env)` stuff at the beginning of this function?